device/mellanox/x86_64-mlnx_msn4600-r0/pcie.yaml

```diff
@@ -0,0 +1,419 @@
+- bus: '00'
```

Comment:
please verify with kebo if this file is OK

Reply:
Talked with Kebo, the file is OK